### PR TITLE
Include other reviewer scores on the performance page (bug 841742)

### PR DESCRIPTION
**Pull request by mstriemer**
https://bugzilla.mozilla.org/show_bug.cgi?id=841742

screenshot 2015-09-08 16 46 35

### CODE REVIEW

File: apps/editors/views.py [hunk 1]
```diff
@@ -222,21 +222,29 @@ def performance(request, user_id=False):
     months = ReviewerScore.get_breakdown_since(user, month_ago)
     years = ReviewerScore.get_breakdown_since(user, year_ago)
 
-    def _sum(iter, types):
-        return sum(s.total for s in iter if s.atype in types)
+    def _sum(iter, types, exclude=False):
+        return sum(s.total
+                   for s in iter
+                   if (s.atype in types) == (not exclude))
```
Comment (magopian) [inline apps/editors/views.py]: Please add a docstring to clearly explain what the `exclude` parameter does

**Comment by magopian** [comment]
r+wc, thanks!

File: apps/editors/views.py (hunk 1 above)
Comment (mstriemer) [inline apps/editors/views.py]: Added a docstring. I'm not crazy about the wording but found it a bit weird to describe.

**Comment by magopian** [comment]
r+, thanks!

**Pull request merged**